packages/app-project/stores/UI.js

```diff
@@ -0,0 +1,26 @@
+import { types } from 'mobx-state-tree'
+
+const UI = types
+  .model('UI', {
+    mode: types.optional(types.enumeration('mode', ['light', 'dark']), 'light')
```

Comment:
Could you note a todo here to check localStorage for a past setting. This is what current PFE uses to persist a light or dark mode choice between sessions.

Reply:
I might set it via a cookie instead. Local storage won’t get queried until the client bundle has loaded, so it’ll switch themes.